### PR TITLE
Add support for custom plugins in ChartJS

### PR DESCRIPTION
**Pull request by malzariey**
## Description

This PR introduces a change in the chart.js where custom plugins can now be integrated. The creation of a new chart now includes an integration of custom plugins if they exist, enhancing the flexibility and customizability of the chart component.

## Visual changes

None

## Functional changes

- [x] Code style has been fixed by running the `composer cs` command.
- [x] Changes have been tested to not break existing functionality.
- [x] Documentation is up-to-date.

### CODE REVIEW

File: packages/widgets/resources/js/components/chart.js
Comment (danharrin) [inline packages/widgets/resources/js/components/chart.js:1]: Can we refactor this somehow to get rid of the one-time variable?

File: packages/widgets/docs/03-charts.md
Comment (danharrin) [inline packages/widgets/docs/03-charts.md:1]: Please use `kebab-case` instead of `snake_case` for file names

**Review by danharrin — CHANGES_REQUESTED** [review]

File: packages/widgets/resources/js/components/chart.js
Comment (malzariey) [reply packages/widgets/resources/js/components/chart.js:1]: Completed.

File: packages/widgets/docs/03-charts.md
Comment (malzariey) [reply packages/widgets/docs/03-charts.md:1]: Completed

**Comment by danharrin** [comment]
Thanks

**Pull request merged**